src/sim/seq.rs

```diff
@@ -185,7 +174,7 @@ where
     /// Kill off phenotypes using stochastic universal sampling.
     fn kill_off(&mut self, count: usize) {
         let ratio = self.population.len() / count;
-        let mut i = ::rand::thread_rng().gen_range::<usize>(0, self.population.len());
+        let mut i = XorShiftRng::new_unseeded().gen_range::<usize>(0, self.population.len());
```

Comment:
It looks like `new_unseeded` is deprecated and you should use the `FromEntropy` or `SeedableRng` trait instead.

Reply:
Yes, I know - fromEntropy won't work with WASM though, I was trying to find something that is sufficiently random and available, but I think it will come back to the user to take care of seeding it properly. This is an architecture issue, I'd say - because PRNGs will repeat after a while which means passing in a RNG is going to different than what it is now (I think).

But in any case, passing in a seed will need an interface, the question is where :smile:

Reply:
I would think that passing in the seed would go well in the `SimBuilder`. You could have a method that sets the seed, and have a default seed as well. That way you should be able to use `Seedable`, not break the interface, and remove this deprecated call 😃